packages/driver/test/cypress/integration/cy/timers_spec.js

```diff
@@ -201,18 +201,45 @@ describe('driver/src/cy/timers', () => {
     })
   })
 
-  it('can use string code as timer function', () => {
-    cy
-    .window()
-    .then((win) => {
-      win.stub = cy.stub()
+  describe('accepts different types of timer function', () => {
+    it('string', () => {
+      cy
+      .window()
+      .then((win) => {
+        win.stub = cy.stub()
 
-      win.setTimeout('this.stub()', 1)
+        win.setTimeout('this.stub()', 1)
 
-      cy
-      .wait(10)
-      .then(() => {
-        expect(win.stub).to.be.called
+        cy
+        .wait(10)
+        .then(() => {
+          expect(win.stub).to.be.called
+        })
+      })
+    })
+
+    const codes = [
+      ['undefined', undefined],
+      ['boolean', true],
+      ['number', 42],
+      ['array', []],
+      ['object', {}],
+    ]
+
+    codes.forEach((code) => {
```

Comment:
I think this is a good place to use destructuring, so you can utilize variable names instead of array indices. e.g.:

```javascript
codes.forEach(([name, value]) => {
  // use name instead of code[0], value instead of code[1]
})
```

Please feel free to use better variables than `name` and `value` if any come to mind. Those are just off the top of my head.